packages/eslint-plugin/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 
 - The `@wordpress/valid-sprintf` rule now detects usage of `sprintf` via `i18n.sprintf` (e.g. when using `import * as i18n from '@wordpress/i18n'`).
 - `@wordpress/no-unused-vars-before-return` will correctly consider other unused variables after encountering an instance of an `excludePattern` option exception.
+- The `@wordpress/no-unused-vars-before-return` rule will now correctly identify valid usage of a variable as a JSX identifier.
```

Comment:
It probably needs to be rebased and moved to the top of the file now.

Reply:
> It probably needs to be rebased and moved to the top of the file now.

👍 Updated in rebased 79d942c4f.